en/includes/guides/authentication/mfa/add-x509-login.md

```diff
@@ -170,22 +204,20 @@ that you need to use later on in this guide.
     ssl_protocol = "TLS"
     ```
 
-    !!! note
+    !!! Important 
     
-        1.  The `clientAuth` attribute causes the Tomcat to require the client with providing a certificate that can be configured as follows.
+        1.   To function properly, this connector should come first in the order. Otherwise, when mutual SSL takes place, the already existing connector (9443) will be picked up and the certificate will not be retrieved correctly.
+
+        2.  The `clientAuth` attribute causes the Tomcat to require the client with providing a certificate that can be configured as follows.
             -   `true` : valid client certificate required for a connection to succeed
             -   `want` : use a certificate if available, but still connect if no certificate is available
             -   `false` : no client certificate is required or validated
     
-        2.   The `truststoreFile` attributes specifies the location of the truststore that contains the trusted certificate issuers.
+        3.   The `truststoreFile` attributes specifies the location of the truststore that contains the trusted certificate issuers.
 
-## Disable certificate validation
+## Step 4: Disable certificate validation
 
-The location that is used to disable certificate validation depends on whether {{ product_name }} was started at least once or not.
-
--   If you have never started {{ product_name }} before, the configurations should be made on the `deployment.toml` file.
-
--   If you have started {{ product_name }} at least once, the configurations should be made on the registry parameters.
+Disabling certificate validation is necessary as {{product_name}} attempts to validate certificates against external Certificate Revocation Lists (CRL) or Online Certificate Status Protocol (OCSP) responders. Since self-signed certificates do not have a publicly accessible CA for revocation checks, enabling these validations will cause authentication failures.
```

Comment:
This is necessary only when using it in not secure scenarios such as self signed CAs. IMO this should be improved to highlight the requirement of enabling/ disabling the cert validation. This way it could be inferred as disabling the cert validation is mandatory requirement for WSO2 IS.